src/components/OpenGovTreasuryProposal/WriteProposal.tsx

```diff
@@ -167,6 +167,14 @@ const WriteProposal = ({
 		setSteps({ percent: 66.6, step: 0 });
 	};
 
+	useEffect(() => {
```

Comment:
no need to do this just set the link in local storage on click on the card and it will take it from there only

Reply:
It doesn't automatically fetch the link on page load which is why I'm using this since the prop is being populated on page load